Sources/Swift/Tools/UrlDetail.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+@objcMembers
+public class UrlDetail: NSObject {
```

Comment:
`m`: I'm a bit confused by the name. Maybe we can rename it to something like
```suggestion
public class UrlSanitized: NSObject {
```